Node/core/lib/bots/ChatConnector.js

```diff
@@ -10,6 +10,7 @@ var jwt = require("jsonwebtoken");
 var zlib = require("zlib");
 var Promise = require("promise");
 var urlJoin = require("url-join");
```

Comment:
See https://github.com/microsoft/BotBuilder-V3/pull/147

These unrelated changes are also part of the above PR. They were made by another dev on April 1, in TS but not in JS. Therefore bringing them in with further TS changes (for example here) is unavoidable.